nucypher/blockchain/eth/interfaces.py

```diff
@@ -210,7 +206,8 @@ def client_version(self) -> str:
             return "Unknown"
 
         if self.is_connected:
-            node_info = self.w3.clientVersion.split(os.sep)
+            node_info = self.w3.clientVersion.split('/')
```

Comment:
do you need to do this, can the technology be obtained from `self.client`? It would be nice to keep that stuff obfuscated within the client

Reply:
Also, I really wanted to rename the `client_version` method on line 204 - it's more of a `client_type` than a `client_version`, what do you think?

Reply:
Yes that should be gone.  As for client_version as long as it isn’t being accessed anywhere I don’t see why not.

Reply:
Yeah I agree - I think the can be forwarded to the web3 client abstraction at `self.client`